lib/src/main/java/com/auth0/jwt/impl/PayloadImpl.java

```diff
@@ -73,14 +76,14 @@ public String getId() {
 
     @Override
     public Claim getClaim(String name) {
-        return extractClaim(name, tree);
+        return extractClaim(name, tree, objectReader);
     }
 
     @Override
     public Map<String, Claim> getClaims() {
-        Map<String, Claim> claims = new HashMap<>();
+        Map<String, Claim> claims = new HashMap<>(tree.size() * 2);
```

Comment:
What's the rationale to pick this default value?

Reply:
Crude estimate of necessary capacity to avoid rehashing and collision. 

https://stackoverflow.com/questions/10901752/what-is-the-significance-of-load-factor-in-hashmap